Lib/http/client.py

```diff
@@ -958,6 +964,21 @@ def _tunnel(self):
         finally:
             response.close()
 
+    def get_proxy_response_headers(self):
```

Comment:
Document this new method in Doc/library/http/client.rst and include a `.. versionadded:: 3.12` marker.